src/bootstrap/test.rs

```diff
@@ -1134,6 +1131,11 @@ help: to skip test's attempt to check tidiness, pass `--exclude src/tools/tidy`
             }
             crate::format::format(&builder, !builder.config.cmd.bless(), &[]);
         }
+
+        builder.info("tidy check");
+        try_run(builder, &mut cmd);
```

Comment:
This change isn't strictly necessary, but I personally find that I have broken formatting way more commonly than any other issue, so it seems nice to run that check first.